pkg/printer/bigtext.go

```diff
@@ -0,0 +1,23 @@
+package printer
+
+import (
+	"github.com/pterm/pterm"
+)
+
+func (p *ConsolePrinter) BigLettersWithColor(colorStr, text string) (string, error) {
```

Comment:
Isn't it better to keep all methods along with the struct definition (in printer/printer.go file)?

Reply:
in this case I agree, will move it